src/Microsoft.Data.SqlClient/netcore/src/Microsoft/Data/SqlClient/TdsParser.cs

```diff
@@ -8411,6 +8411,21 @@ internal int WriteSQLDNSCachingFeatureRequest(bool write /* if false just calcul
             return len;
         }
 
+        internal int WriteJsonSupportFeatureRequest(bool write /* if false just calculates the length */)
```

Comment:
Are there any unit tests that cover these parser methods?

Reply:
From what I understand, there are no separate unit tests for these methods. We have added the processing of these feature request in the TDS test server.